prdoc/pr_4527.prdoc

```diff
@@ -0,0 +1,27 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Call implementation for `transfer_all`
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR introduces the `transfer_all` call for `pallet-assets`.
+      The parameters are analog to the same call in `pallet-balances`.
+      This change shouldn't break the existing APIs.
```

Comment:
new call is new enum variant within Call enum. so this is a breaking change as for a rust crate.

Reply:
That's rough. I guess we have to follow crates rules but it's not breaking for transaction encoding.

Reply:
Yes, but for crates versioning I think it should follow the rust semver rules. At least that's what we have not in guidelines.

Reply:
Yeah, @ggwpez also pointed out that it breaks the `WeightInfo` trait for anyone using this pallet. So I agree it is (unfortunately) major.